File: golang/model/ads_feature.go
package model

// 広告案件に紐づく特徴量を保存するクラス。
type AdsFeature struct {
	AdsId   uint `db:"ads_id"`  // 広告案件ID
	Feature int  `db:"feature"` // 広告に紐づく特徴量
}

File: python/model/ads_feature.py
from dataclasses import dataclass


@dataclass(frozen=True)
class AdsFeature:
    """広告案件に紐づく特徴量を保存するクラス"""

    ads_id: int  # 広告案件ID
    feature: int  # 広告に紐づく特徴量
